libcontainer/rootfs_linux.go

```diff
@@ -464,6 +464,30 @@ func doBindMounts(config *configs.Config, pipe io.ReadWriter) error {
 	return nil
 }
 
+func chownMounts(config *configs.Config, pipe io.ReadWriter, chownList []string) error {
+	chownReqs := []opReq{}
+
+	for _, path := range chownList {
+		req := opReq{
+			Op:     chown,
+			Rootfs: config.Rootfs,
+			Path:   path,
+			Uid:    config.UidMappings[0].HostID,
+			Gid:    config.GidMappings[0].HostID,
+		}
+
+		chownReqs = append(chownReqs, req)
+	}
+
+	if len(chownReqs) > 0 {
+		if err := syncParentDoOp(chownReqs, pipe); err != nil {
+			return newSystemErrorWithCause(err, "syncing with parent runc to perform bind mounts")
```

Comment:
Adjust error msg description: "to perform bind mounts".

Reply:
Done, thanks!